selfdrive/car/honda/interface.py

```diff
@@ -200,6 +200,19 @@ def get_params(candidate, fingerprint):
       ret.sR = 15.3
       ret.steerKp, ret.steerKi = 0.8, 0.24
 
+      ret.longitudinalKpBP = [0., 5., 35.]
+      ret.longitudinalKpV = [1.2, 0.8, 0.5]
```

Comment:
Is the brake system like the one on the civic or like the one on the acura ILX you think? Can you hear constant pump noise when brakes are applied at low speed? If so, it's probably like the acura and this tuning is fine.